week2/4_Lists.ipynb

```diff
@@ -341,7 +341,8 @@
     "    </div>\n",
     "    <div style=\"width: 90%\">\n",
     "        <p style=\"text-align: right; direction: rtl; clear: both;\">\n",
-    "            המספור של התא הראשון הוא 0, ולא 1. יש לכך סיבות טובות, אבל פעמים רבות זה ירגיש לא טבעי ועלול ליצור באגים.<br>\n",
+    "            המספור של התא הראשון הוא 0, ולא 1. יש לכך סיבות טובות, אבל פעמים רבות זה ירגיש לא טבעי ועלול ליצור <dfn>באגים</dfn>.<br>\n",
+    "            <em>* באג הוא קטע קוד המתנהג לא כמצופה.</em><br>\n",
```

Comment:
Merge with the line about באגים, so it would sound more natural.

המספור של התא הראשון הוא 0, ולא 1. יש לכך סיבות טובות, אבל פעמים רבות זה ירגיש לא טבעי.<br>
בלבול של המתכנת בעקבות ההתנהגות עלול לגרום ל\<dfn>באגים\<dfn>, קטעי קוד שמתנהגים אחרת משציפה המתכנת.